src/svg-icons/delete.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
```

Comment:
Added some icons from master, but i suppose they will be transferred all at some point?